.github/workflows/main-workflow.yml

```diff
@@ -18,14 +18,14 @@ jobs:
     - name: tshirts fail
       run: |
         cd tshirts
-        bash ../.github/workflows/expectfail.sh java -ea tshirts.java

```

Comment:
Removing this will miss the running of the tests. Retain ./tshirts.out in the workflow. Same for the lines below as well